common/array.h

```diff
@@ -330,13 +350,15 @@ class Array {
 		}
 	}
 
+    /** Change the size of the array. */
 	void resize(size_type newSize) {
 		reserve(newSize);
 		for (size_type i = _size; i < newSize; ++i)
 			new ((void *)&_storage[i]) T();
 		_size = newSize;
 	}
 
+    /** @todo Doc required. */
```

Comment:
Assign to this array the elements between the given iterators (from @p first included to @p last excluded) from another array.